ludwig/schema/trainer.py

```diff
@@ -753,12 +768,38 @@ class LLMTrainerConfig(BaseTrainerConfig):
 
 
 @DeveloperAPI
-@register_trainer_schema(MODEL_LLM)
+@register_llm_trainer_schema("zeroshot")
 @ludwig_dataclass
 class ZeroShotTrainerConfig(LLMTrainerConfig):
     """Dataclass that configures most of the hyperparameters used for zero-shot LLM model training."""
 
-    pass
+    # Required for lookup during trainer initialization
+    type: str = schema_utils.ProtectedString("zeroshot")
+
+
+@DeveloperAPI
+@register_llm_trainer_schema("fewshot")
```

Comment:
Another follow-up: we could just have a single trainer, call it "PassthroughTrainer" or something, and then have it used by default when no trainer is specified.

Reply:
Agreed, will add this in a follow-up PR